R/shinywrappers.R

```diff
@@ -53,6 +53,12 @@ markRenderFunction <- function(
   cacheWriteHook = NULL,
   cacheReadHook = NULL
 ) {
+  # # Do not include this code
+  # # This method is called by the superseding function, createRenderFunction().
```

Comment:
If you want to include this, let's expand on the details of why this func is "sort of" superceded but not actually